usr/share/rear/setup/default/007_set_dmesg_level.sh

```diff
@@ -0,0 +1,39 @@
+
+# This file is part of Relax-and-Recover, licensed under the GNU General
+# Public License. Refer to the included COPYING for full text of license.
+
+# In the ReaR recovery system /etc/scripts/boot calls hardcoded 'dmesg -n 5'
+# to limit console logging for kernel messages to level 5 during recovery system startup
+# (the usual default shows too many kernel messages that disturb the intended ReaR messages)
+# so that kernel error and warning messages appear (intermixed with ReaR messages) on the console
+# so the user can notice when things go wrong in kernel area which helps to understand problems,
+# see https://github.com/rear/rear/issues/3107 and https://github.com/rear/rear/pull/3108
+
+# Additionally call 'dmesg -n [4-6]' here depending on verbose and debug modes for ReaR
+# if we are inside the ReaR recovery system (we must not change things on the original system),
+# see https://github.com/rear/rear/issues/3107#issuecomment-1855797222
+test "$RECOVERY_MODE" || return 0
+
+# Now we are inside the ReaR recovery system
+# and 'setup' stage scripts are only run by the following workflows
+# recover layoutonly restoreonly finalizeonly mountonly
+# see https://github.com/rear/rear/pull/3112#issuecomment-1862770147
+# so for those workflows 'dmesg -n [4-6]' is set depending on verbose and debug modes for ReaR.
+
+# Set minimum dmesg level to 4 to show at least kernel error conditions and more severe issues on the console
+dmesg_level=4
```

Comment:
Yes, of course.

Fixed via
https://github.com/rear/rear/pull/3113/commits/830b11e3896bea3b4508cf8141f2dcb8cc4d6919

It was missing because of copy&paste from
https://github.com/rear/rear/pull/3112/files
(there cannot be 'local' in sbin/rear).